File: 4-MedianOfTwoSortedArrays/4-MedianOfTwoSortedArrays.py
# Last updated: 7/31/2025, 5:38:52 PM
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        a=sorted(list(nums1+nums2))
        if len(a)%2!=0:
            return (float(a[(len(a))//2]))
        else:
            p=a[(len(a))//2]
            
            g=a[((len(a))//2)-1]
            return float((p+g)/2)
        

File: 4-MedianofTwoSortedArrays/4-MedianofTwoSortedArrays.java
// Last updated: 12/26/2025, 2:44:53 PM
1class Solution {
2    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
3        int[] num = new int[nums1.length+nums2.length];
4        for(int i=0; i<nums1.length; i++){
5            num[i]=nums1[i];
6        }
7        int c = nums1.length;
8        for(int i=0; i<nums2.length; i++){
9            num[c]=nums2[i];
10            c++;
11        }
12        Arrays.sort(num);
13        if(num.length%2!=0){
14            double a = num[num.length/2];
15            return a;
16        }
17        double a = num[num.length/2];
18        double b = num[(num.length-1)/2];
19        return (a+b)/2.0;
20
21        
22    }
23}